site/getting-started/introduction.md

```diff
@@ -20,20 +20,16 @@ head:
 
 # Quick start
 
-This guide will show you how to create Embedded Accounts using Account Kit, the Alchemy Signer, and the Alchemy Modular Smart Contract Account.
+This guide will show you how to create Embedded Accounts using [Account Kit](https://accountkit.alchemy.com/resources/terms.html#account-kit), the Alchemy [Signer](https://accountkit.alchemy.com/resources/terms.html#signer), and the Alchemy [Modular Smart Contract Account](https://accountkit.alchemy.com/resources/terms.html#modular-account).
```

Comment:
```suggestion
This guide will show you how to create Embedded Accounts using [Account Kit](https://accountkit.alchemy.com/resources/terms.html#account-kit), the Alchemy [Signer](https://accountkit.alchemy.com/resources/terms.html#signer), and the Alchemy [Modular Account](https://accountkit.alchemy.com/resources/terms.html#modular-account).
```

Reply:
we're just calling it MA everywhere now so we should do a pass and make sure that holds in our docs as well